Parse/Internal/Analytics/Controller/PFAnalyticsController.m

```diff
@@ -44,15 +50,19 @@ + (instancetype)controllerWithEventuallyQueue:(PFEventuallyQueue *)eventuallyQue
 
 - (BFTask *)trackAppOpenedEventAsyncWithRemoteNotificationPayload:(NSDictionary *)payload
                                                      sessionToken:(NSString *)sessionToken {
-    // If the Remote Notification payload had a message sent along with it, make
-    // sure to send that along so the server can identify "app opened from push"
-    // instead.
-    id alert = payload[@"aps"][@"alert"];
-    NSString *pushDigest = (alert ? [PFAnalyticsUtilities md5DigestFromPushPayload:alert] : nil);
-
-    PFRESTCommand *command = [PFRESTAnalyticsCommand trackAppOpenedEventCommandWithPushHash:pushDigest
-                                                                               sessionToken:sessionToken];
-    return [[self.eventuallyQueue enqueueCommandInBackground:command] continueWithSuccessResult:@YES];
+    @weakify(self);
+    return [[BFTask taskFromExecutor:[BFExecutor defaultPriorityBackgroundExecutor] withBlock:^id{
+        @strongify(self);
+        // If the Remote Notification payload had a message sent along with it, make
+        // sure to send that along so the server can identify "app opened from push"
+        // instead.
+        id alert = payload[@"aps"][@"alert"];
+        NSString *pushDigest = (alert ? [PFAnalyticsUtilities md5DigestFromPushPayload:alert] : nil);
+
+        PFRESTCommand *command = [PFRESTAnalyticsCommand trackAppOpenedEventCommandWithPushHash:pushDigest
+                                                                                   sessionToken:sessionToken];
+        return [self.eventuallyQueue enqueueCommandInBackground:command];
+    }] continueWithSuccessResult:@YES];
```

Comment:
Since the result of this operation doesn't matter here, only the error - we are chaining on the task to return `NSNumber numberWithBool:YES` simply to indicate the success of the operation.

`continuWithSuccessResult:` is a simple method wrapper and is identical to
```objc
continueWithSuccessBlock:^id(BFTask *task) {
  return @YES;
}]
```